src/Ombi.Core/Authentication/OmbiUserManager.cs

```diff
@@ -61,14 +64,46 @@ public class OmbiUserManager : UserManager<OmbiUser>
             _embySettings = embySettings;
             _jellyfinSettings = jellyfinSettings;
             _authSettings = auth;
+            _ldapUserManager = ldapUserManager;
+            _userManagementSettings = userManagementSettings;
         }
 
         private readonly IPlexApi _plexApi;
         private readonly IEmbyApiFactory _embyApi;
         private readonly IJellyfinApiFactory _jellyfinApi;
+        private readonly ILdapUserManager _ldapUserManager;
         private readonly ISettingsService<EmbySettings> _embySettings;
         private readonly ISettingsService<JellyfinSettings> _jellyfinSettings;
         private readonly ISettingsService<AuthenticationSettings> _authSettings;
+        private readonly ISettingsService<UserManagementSettings> _userManagementSettings;
+
+        public async Task<OmbiUser> FindUser(string userName)
```

Comment:
I'm not sure about this method, It has mixed responsibilities, and it's not just FindUser. It also could potentially create the user.